beast/observationmodel/ast/make_ast_xy_list.py

```diff
@@ -336,43 +366,54 @@ def pick_positions_from_map(
                     if catalog_boundary_radec:
                         within_bounds = catalog_boundary_radec.contains_points(
                             [[x, y]]
-                        )[0]  # N,2 array of AST X and Y positions
+                        )[
+                            0
+                        ]  # N,2 array of AST X and Y positions
                         if not within_bounds:
                             x = None
 
                     # check that this x/y is with any input boundary
                     if set_coord_boundary is not None:
                         if coord_boundary_radec:
-                            within_bounds = coord_boundary_radec.contains_points([[x, y]])[0]
+                            within_bounds = coord_boundary_radec.contains_points(
+                                [[x, y]]
+                            )[0]
                             if not within_bounds:
                                 x = None
                     if region_from_filters is not None:
                         if filt_reg_boundary_radec:
-                            within_bounds = filt_reg_boundary_radec.contains_points([[x, y]])[0]
+                            within_bounds = filt_reg_boundary_radec.contains_points(
+                                [[x, y]]
+                            )[0]
                             if not within_bounds:
                                 x = None
 
-
                 # if we can convert to x/y, do everything in x/y
                 else:
-                    [x], [y] = wcs.all_world2pix(np.array([ra]), np.array([dec]), wcs_origin)
+                    [x], [y] = wcs.all_world2pix(
+                        np.array([ra]), np.array([dec]), wcs_origin
+                    )
 
                     # check that this x/y is within the catalog footprint
-                    within_bounds = catalog_boundary_xy.contains_points(
-                        [[x, y]]
-                    )[0]  # N,2 array of AST X and Y positions
+                    within_bounds = catalog_boundary_xy.contains_points([[x, y]])[
+                        0
```

Comment:
I agree.  I'm assuming it's just a strange outcome from blackening the code?

Reply:
Yep.  I'll investigate and turn off the formatting for this line if I can't solve it some other way.

Reply:
This auto-formatting happens for a number of lines due to the long variable names and how the tuple results are unpacked.  Instead of adding comment lines for each case to turn the formatting on/off, I just made the destination variable name `inbounds` instead of `within_bounds`.  This solves the problem and the variable name still means the same thing (I would posit).